inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -519,8 +520,22 @@ std::shared_ptr<ngraph::Node> V10Parser::createNode(const std::vector<ngraph::Ou
             ngraphNode->constructor_validate_and_infer_types();
     }
 
+    // Check that operation is ExperimentalDetectron*
+    std::string type = params.type;
+    const std::set<std::string> experimental_detectrons = {"ExperimentalDetectronDetectionOutput"};
+    bool isDetectron = experimental_detectrons.count(type) != 0;
+    if (!ngraphNode && params.version == "experimental" && isDetectron) {
+        auto opset = opsets.at("opset6");
+        ngraphNode = std::shared_ptr<ngraph::Node>(opset.create(type));
+        ngraphNode->set_friendly_name(params.name);
+        ngraphNode->set_arguments(inputs);
+        XmlDeserializer visitor(node, weights);
+        if (ngraphNode->visit_attributes(visitor))
+            ngraphNode->constructor_validate_and_infer_types();
+    }
+
     // Create GenericIE operation for backward compatibility
-    if (!ngraphNode && (params.version == "experimental" || params.version == "extension")) {
+    if (!ngraphNode && !isDetectron && (params.version == "experimental" || params.version == "extension")) {
```

Comment:
`isDetectron` is redundant, because `ngraphNode` already will be created if it is a detectron.

Reply:
Fixed.